.env.example

```
@@ -32,12 +32,12 @@ REDIS_PORT=6379
 
 MAIL_DRIVER=smtp
 MAIL_MAILER=smtp
-MAIL_HOST=smtp.gmail.com
-MAIL_PORT=587
-MAIL_USERNAME=
-MAIL_PASSWORD=
-MAIL_ENCRYPTION=tls
-MAIL_FROM_ADDRESS=
+MAIL_HOST=localhost
```

Do not change these lines. By default i think it should be pointed to gmail instead "nothing"

---

Please revert back to default with my email soulevilx@gmail.com and all related. Except password